client/src/views/EventsView.vue

```diff
@@ -29,8 +29,15 @@
         :isTeamAdmin="tempIsTeamAdmin"
         @edit="openEditModal"
       />
+      <p
+        v-if="filteredEventsList.length == 0"
+        class="pt-6 mx-3 text-center font-weight-bold text-body-1"
+      >
+        No current events :(
+      </p>
     </div>
   </div>
+  <p v-else class="pt-6 mx-3">Loading...</p>
```

Comment:
Something like this could be cool :)
https://vuetifyjs.com/en/components/progress-circular/